auth/customers_test.go

```diff
@@ -0,0 +1,46 @@
+package auth
+
+import (
+	"os"
+	"testing"
+)
+
+func Test_findCustomerURL(t *testing.T) {
+
+	var URIOpts = []struct {
+		title       string
+		envVar      string
+		expectedURL string
+	}{
+		{
+			title:       "No ENV var",
+			envVar:      "",
+			expectedURL: defaultCustomersURL,
+		},
+		{
+			title:       "ENV var exists",
+			envVar:      "https://raw.githubusercontent.com/rgee0/derek/master/.CUSTOMERS",
+			expectedURL: "https://raw.githubusercontent.com/rgee0/derek/master/.CUSTOMERS",
+		},
+		{
+			title:       "Invalid ENV var exists",
+			envVar:      "raw.githubusercontent.com/rgee0/derek/master/.CUSTOMERS",
+			expectedURL: "https://raw.githubusercontent.com/rgee0/derek/master/.CUSTOMERS",
+		},
+	}
+
+	for _, test := range URIOpts {
```

Comment:
I won't be picky but I think go likes these things to be written like `uriOpts`

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
From linked page:

> For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"

The linked page implies both are valid. The current codebase uses the first variant. Doesn't need to change